tutorials/034 - Distributed Calls on Remote Cluster.ipynb

```diff
@@ -0,0 +1,186 @@
+    "\n",
+    "#### Configure Ray Cluster Configuration\n",
+    "Start with a cluster configuration file (YAML)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "touch config.yml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Replace all values to match your desired region, account number and name of resources deployed by the above CloudFormation Stack.\n",
+    "\n",
+    "[Click here](https://console.aws.amazon.com/ec2/home?region=us-east-1#Images:visibility=public-images;search=:ray-amzn-wheels_latest_amzn_ray-1.9.2-cp38;v=3;$case=tags:false%5C,client:false;$regex=tags:false%5C,client:false) to find the Ray AMI for your desired region. The example configuration below uses the AMI for `us-east-1`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cluster_name: pandas-sdk-dist-testing\n",
+    "\n",
+    "initial_workers: 2\n",
+    "min_workers: 2\n",
+    "max_workers: 2\n",
+    "\n",
+    "provider:\n",
+    "    type: aws\n",
+    "    region: us-east-1 # Change AWS region as necessary\n",
+    "    availability_zone: us-east-1a,us-east-1b,us-east-1c # Change as necessary\n",
+    "    security_group:\n",
+    "        GroupName: ray-cluster\n",
+    "    cache_stopped_nodes: False\n",
+    "\n",
+    "available_node_types:\n",
+    "  ray.head.default:\n",
+    "    node_config:\n",
+    "      InstanceType: m5n.xlarge\n",
+    "      IamInstanceProfile:\n",
+    "        # Replace with your account id and profile name if you did not use the default value\n",
+    "        Arn: arn:aws:iam::{ACCOUNT ID}:instance-profile/ray-cluster\n",
+    "      ImageId: ami-0ea510fcb67686b48\n",
+    "\n",
+    "  ray.worker.default:\n",
+    "      min_workers: 2\n",
+    "      max_workers: 2\n",
+    "      node_config:\n",
+    "        InstanceType: m5n.xlarge\n",
+    "        IamInstanceProfile:\n",
+    "          # Replace with your account id and profile name if you did not use the default value\n",
+    "          Arn: arn:aws:iam::{ACCOUNT ID}:instance-profile/ray-cluster\n",
+    "        ImageId: ami-0ea510fcb67686b48\n",
+    "\n",
+    "\n",
+    "setup_commands:\n",
+    "- pip install \"awswrangler[distributed]==3.0.0a2\"\n",
+    "- export AWS_DEFAULT_REGION=us-east-1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can now execute actions. The SDK will detect the required dependencies for its `distributed` mode and parallelize the methods that support it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import awswrangler as wr\n",
```

Comment:
One of two things must happen here:
1. The user connects to the cluster by submitting the address (import ray then ray.init(address=...) # ec2 head node address
2. The user submits their script to the cluster with `ray submit`

I prefer #1 but we must obtain the address somehow and test this from a notebook

Reply:
We would also need a cell to actually create the cluster first

Reply:
@jaidisido How does L112 - L137 look??

This is the approach I've used in testing that's worked locally. I will test this from a notebook as well.

Reply:
Notebook test is working.

Reply:
Nice! Thanks